src/powershell/Public/Remove-FinOpsCostExport.ps1

```diff
@@ -49,121 +42,83 @@ function Remove-FinOpsCostExport
+      ResourceType         = "exports"
+      Name                 = $Name
+      ApiVersion           = $ApiVersion
+      Payload              = $payload
+    }
+
+    $httpResponse = Invoke-AzRestMethod @invokeAzRestMethodParams -Method "GET"
 
-    if ($httpResponse.StatusCode -eq 200)
-    {
-      # Export deleted successfully
+    if ($httpResponse.StatusCode -eq 404) {
+      Write-Verbose -Message "Cost Management export not found."
+      break
     }
-    elseif ($httpResponse.StatusCode -eq 404)
+    elseif ($httpResponse.StatusCode -ne 200)
     {
-      # Not found
+      throw ($script:localizedData.GetCostExportNotFound -f $($httpResponse.Content))
     }
     else
     {
-      # Error response describing why the operation failed.
-      throw "Not Exists: Cost Management Export operation failed with message: '$($httpResponse.Content)'"
-    }
-
-    if ($PSCmdlet.ShouldProcess($Name, 'DeleteCostExport'))
-    {
-
-      # Using the REST API to delete the export as requested as PS modules are outdated?
-      $httpResponse = Invoke-AzRestMethod `
-        -ResourceProviderName "Microsoft.CostManagement" `
-        -ResourceType "exports" `
-        -Name $Name `
-        -ApiVersion $ApiVersion `
-        -Method "DELETE" `
-        -Payload $payload
-
-      if ($httpResponse.StatusCode -eq 200)
-      {
-        # Export deleted successfully
-      }
-      elseif ($httpResponse.StatusCode -eq 404)
-      {
-        # Not found - nothing to delete
-      }
-      else
-      {
-        # Error response describing why the operation failed.
-        throw "Delete Cost Management Export operation failed with message: '$($httpResponse.Content)'"
-      }
-    }
-
-    # Delete associated ingestion data from storage account
-    if ($RemoveData)
-    {
-      # Using the REST API to get the export as requested as PS modules are outdated?
-      $httpResponse = Invoke-AzRestMethod `
-        -ResourceProviderName "Microsoft.CostManagement" `
-        -ResourceType "exports" `
-        -Name $Name `
-        -ApiVersion $ApiVersion `
-        -Method "GET" `
-        -Payload $payload
-
-      if ($httpResponse.StatusCode -eq 200)
+      # Delete associated data from storage account
+      if ($RemoveData)
       {
-
         # Export details retreived
         $exportDetails = ConvertFrom-Json -InputObject $httpResponse.Content
-        $storageAccountID = $exportDetails.properties.deliveryInfo.destination.resourceId
-
-        # Get-AzStorageAccount -resourcegroupname alz-finopstk -name finopshub255k7ov6asado | Get-AzDataLakeGen2ChildItem -FileSystem "ingestion" -Path "/" -Recurse -FetchProperty | Where-Object IsDirectory -ne false | Select-Object Path
+        $storageAccountId = $exportDetails.properties.deliveryInfo.destination.resourceId
         $resourceGroupName = $storageAccountID.Split('/')[4]
         $storageAccountName = $storageAccountID.Split('/')[8]
+        $path = $scope + "/" + $Name
```

Comment:
Use the path associated with the export since we need to support all exports, not just those connected to hubs.
```suggestion
        $path = $exportDetails.properties.deliveryInfo.destination.rootFolderPath + "/" + $Name
```